.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think some options are the same default value, so not need to define theme again.

Like:
```json
{
  "semi": true,
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "arrowParens": "avoid"
}
```

Reply:
May check http://json.schemastore.org/prettierrc

Reply:
I didn't notice that, will change it next patch